util/src/test/java/io/kubernetes/client/informer/impl/DefaultSharedIndexInformerTest.java

```diff
@@ -12,366 +12,47 @@
 */
 package io.kubernetes.client.informer.impl;
```

Comment:
Renamed DefautSharedInformterTest to DefaultSharedIndexInformerWireMockTest and added a pure unit test DefautSharedInformterTest. (It is not as big as it looks like :) )